cg/models/devices/illumina_run_directory_data.py

```diff
@@ -317,17 +319,17 @@ def __str__(self):
         return f"FlowCell(path={self.path},run_parameters_path={self.run_parameters_path})"
 
 
-def get_flow_cells_from_path(flow_cells_dir: Path) -> list[FlowCellDirectoryData]:
-    """Return flow cell objects from flow cell dir."""
-    flow_cells: list[FlowCellDirectoryData] = []
-    LOG.debug(f"Search for flow cells ready to encrypt in {flow_cells_dir}")
-    for flow_cell_dir in flow_cells_dir.iterdir():
-        if not flow_cell_dir.is_dir():
+def get_sequencing_runs_from_path(sequencing_run_dir: Path) -> list[IlluminaRunDirectoryData]:
+    """Return sequencing run objects from sequencing run dir."""
+    sequencing_runs: list[IlluminaRunDirectoryData] = []
+    LOG.debug(f"Search for flow cells ready to encrypt in {sequencing_run_dir}")
+    for run_dir in sequencing_run_dir.iterdir():
+        if not run_dir.is_dir():
             continue
-        LOG.debug(f"Found directory: {flow_cell_dir}")
+        LOG.debug(f"Found directory: {run_dir}")
         try:
-            flow_cell = FlowCellDirectoryData(flow_cell_path=flow_cell_dir)
+            sequencing_run = IlluminaRunDirectoryData(sequencing_run_path=run_dir)
         except FlowCellError:
             continue
-        flow_cells.append(flow_cell)
-    return flow_cells
+        sequencing_runs.append(sequencing_run)
+    return sequencing_runs
```

Comment:
Just in general for this class, I do not see the benefit from renaming flow cell to sequencing run. Is sequencing run more specific for Illumina or are we not losing information in logs, comments and method names here?

Reply:
I guess for the method names the class name might be enough though.

Reply:
sequencing run is more specific to what it actually represents. The use of flow cell as an output directory is a bit misleading since you just get data that was generated by a sequencing run that consumed a flow cell.